index.html

```diff
@@ -0,0 +1,119 @@
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Program</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Join</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Sponsor</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5 me-3">News</li>
+                        </a>
+                        <a style="border: 3px solid rgb(255, 51, 0); border-radius: 5px;color: rgb(255, 51, 0);"
+                            href="#">
+                            <li class="mx-5">Campaign</li>
+                        </a>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <div class="container text-dark sm-fs-4">
+            <h2 class="pt-3 pb-3 mt-1" style="color: rgb(255, 51, 0);">"Hello, Sharing world"</h2>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID <br><br><br> ARTS FESTIVAL<br><br><br> 2022</h1><br>
+            <p class="bg-light p-1">Container Festival is without doubt the Bucaramanga's most famous music and
+                performing arts festival, if not the
+                world's. Embracing every genre across the spectrum </p>
+            <span class="date">2022.11.03(THU) ~ 06(SAT)</span><br>
+            <small>@National Museum, Modern Art Museum and more</small>
+        </div>
+    </div>
+
+    <div class="programwrapper">
+        <h1 class="text-center fs-3 " style="color: #fff;">Main Program</h1>
+        <hr class='mb-4'
+            style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 25%; margin-left: 37%;">
+        <div class="mainprogram">
+        </div>
+        <div class="wholeprogrambutton"></div>
+    </div>
+
+    <div class="speakerswrapper">
+        <h1 class="text-center fs-3 mt-5" style="color: rgb(0, 0, 0);">Speakers</h1>
+        <hr class="mb-5" style="color:rgb(255, 51, 0) ;border: 2px solid rgb(255, 51, 0);width: 25%; margin-left: 37%;">
+        <div class="speakers">
+        </div>
+    </div>
+
+    <div class="open">
+        <div class="close">
+            <a href="#"> <img src="img/Icon.png" alt="x icon"></a>
+        </div>
+    </div>
+
+    <div class="sponsordiv">
+        <h1 class="text-center fs-3 mt-4" style="color: rgb(255, 255, 255);">Partners</h1>
+        <hr style="color:rgb(255, 51, 0) ;border: 2px solid rgb(255, 51, 0);width: 25%; margin-left: 37%;">
+
+        <div class="d-flex justify-content-center sponsor">
+            <img src="img/firefox.svg" alt="logo google play">
+            <img src="img/google-2.svg" alt="logo youtube">
+            <img src="img/cc-square-fill.svg" alt="logo chrome">
+            <img src="img/android2.svg" alt="logo android">
+            <img src="img/airbnb.svg" alt="logo android 2">
+        </div>
```

Comment:
- [OPTIONAL] the same thing here ⬆,  I think it would be better to use sematic HTML5 tags like `<section> ` instead of  `<div>` to follow the best practice and Improve the SEO 💯